sdk/servicebus/azure-servicebus/samples/async_samples/mgmt_queue_async.py

```diff
@@ -27,7 +28,10 @@
 
 async def create_queue(servicebus_mgmt_client):
     print("-- Create Queue")
-    await servicebus_mgmt_client.create_queue(QUEUE_NAME, max_delivery_count=10, dead_lettering_on_message_expiration=True)
+    try:
+        await servicebus_mgmt_client.create_queue(QUEUE_NAME, max_delivery_count=10, dead_lettering_on_message_expiration=True)
+    except ResourceExistsError:
+        pass
```

Comment:
my gut feeling is that samples should be kept simple and intuitive.

but through this way we're introducing new mental burden to users -- exception handling and `azure.core.exceptions.ResourceExistsError` just for fixing our livetest..

besides, I'm wondering whether the fix would work because based on the description in the issue:
> This results in the same error randomly, even when the resource does not already exist.

the resources not being created yet will also lead to this `ResourceExistsError` error. Will the update/delete operation fail because the entity doesn't exist?

Reply:
I'm wondering if it's just the mgmt_rule(_async) causing the random exist failure, because in rule sample we truncate uuid4()[0:16]. (there's limitation on entity name length).


probably we could try:
```
RULE_WITH_SQL_FILTER_NAME = "sql_rule" + str(uuid.uuid4())
```

Reply:
I did notice it was primarily, if not always, the mgmt_rule samples. I'll try that, thanks!